### DISCUSSION TITLE
Problems with Data and GIS

### DISCUSSION THREAD

**Discussion by skarakas1** [albertkun/21S-ASIAAM-191A#17] [Past Thinking Caps]
Data science and GIS are incredible tools that open new possibilities for research, analysis, and visualization. One can construct maps at home with freely available satellite, remote sensing, and census data without surveying, using drafting tools, or physically digging through archives. The accessibility of these tools would have been unthinkable not long ago. However, that same accessibility and convenience means that those tools can be more easily deployed for misinformation. As with many facets of the internet, increased accessibility means increased content and less gatekeeping--but also increased responsibility to fact check and think critically.
Mapping inscribes power. The border between California and Mexico divides indigenous Kumeyaay land and impedes the free movement of the tribe on their own land, but few maps of the area will contain that information, constituting an erasure of the region as it would be described by its original inhabitants. Decisions about what to include or not can reproduce, reveal, or confront power asymmetries.
Data, too, must be interrogated. Choropleth maps frequently use data from the US Census Bureau, but Census data is far from perfect. Census questions conflate race and ethnicity and may be subject to volunteer bias or exclusion bias. France and Germany don't collect demographic data on ethnicity at all, making it extraordinarily difficult to statistically describe structural inequality. World Bank international development data is also flawed: the "country" unit obscures regional difference and the concept of development is founded on Eurocentric ideals and used to justify neoliberal extractivism.
Scale is important. A world map of infant and maternal mortality by country would obscure the racial disparities of that statistic within the US, for example.
We live in an age of overabundant information. What we do with it may reflect our own implicit biases and those of sources used. By questioning, critiquing, and collaborating we may be able to find meaningful patterns and represent them more honestly.

**Comment by rachan2023** [comment]
I agree with the points that you made! Data and GIS are important for presenting information to the public, but sometimes it can forgotten that people are collecting the data and creating maps and data visualizations, which can be subject to error and biases. As you mentioned above, it is necessary to think critically about data visualizations and maps. For example, with the erasure of the Kumeyaay land on maps, we have to ask ourselves why did the mapmakers erase their land from the map, what narrative are the mapmakers trying to convey, and what are the implications of this erasure on the Kumeyaay tribe. Overall, data and GIS are very useful tools, but we have to question the data visualizations that we are presented and understand the consequences when these tools are used, as you've mentioned above.
